redis/Cargo.toml

```diff
@@ -63,6 +63,9 @@ crc16 = { version = "0.4", optional = true }
 rand = { version = "0.8", optional = true }
 derivative = { version = "2.2.0", optional = true }
 
+# Only needed for async cluster
+dashmap = { version = "6.0.1", optional = true }
```

Comment:
why are we locking into a a specific version? why not `6.0` ?

Reply:
It doesn't matter, it still means 6.0.1 and above. but I can change it

Reply:
see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Reply:
I am ok with this, I wondered why the ".1" does it contain a fix or was it just the latest at the time picked the crate ...